airgun/widgets.py

```diff
@@ -1588,17 +1588,17 @@ class Pagination(Widget):
     and current page index/overall amount of pages. Mainly used with Table widget.
     """
 
-    ROOT = ".//form[contains(@class, 'content-view-pf-pagination')]"
+    ROOT = ".//foreman-react-component[contains(@name, 'Pagination')]"
     # Kattelo views use per_page with select, foreman use a per_page with Button DropDown.
     PER_PAGE_BUTTON_DROPDOWN = ".//div[button[@id='pagination-row-dropdown']]"
```

Comment:
```suggestion
    PER_PAGE_BUTTON_DROPDOWN = ".//div[ul[@class='pf-c-options-menu__menu']]"
```
This is now represented as an unordered list with class `pf-c-options-menu__menu`, and each menu option is a list item with e.g. `data-action="per-page-5"`. I don't think it's necessary to add locators for the menu options, but this suggestion should make it easier to identify what needs to be added if someone needs to access those elements in the future.